app/controllers/public_recipes_controller.rb

```diff
@@ -0,0 +1,16 @@
+class PublicRecipesController < ApplicationController
+  # skip_before_action :authenticate_user!, only: [:index]
+  def index
+    @public_recipe_data = []
+    public_recipes = Recipe.includes(%i[foods recipe_foods user]).where(public: true).order('created_at DESC')
+    public_recipes.each do |recipe|
```

Comment:
I created two public recipes.

![image](https://user-images.githubusercontent.com/38088818/181827237-f25c4371-326f-451d-bb21-109008319de3.png)

Yet, when I navigate to home where you are listing your public recipes nothing is displayed.

![image](https://user-images.githubusercontent.com/38088818/181827435-f418ba88-310f-4a11-8ee2-7c55fd537cac.png)

- Kindly make sure that public recipes are listed as per project requirements:

![image](https://user-images.githubusercontent.com/38088818/181827728-6424275d-640e-4e4f-a304-859fdc2798a2.png)

![image](https://user-images.githubusercontent.com/38088818/181827950-49485892-88e4-43ae-a1a5-4132be129485.png)